src/serde_impl/mod.rs

```diff
@@ -3,3 +3,4 @@ pub(crate) mod range;
 pub(crate) mod unix_timestamp_micro_secs;
 pub(crate) mod unix_timestamp_secs;
 pub(crate) mod signature_cipher;
+pub(crate) mod date;
```

Comment:
Maybe make the name more verbose like `date_ymd`, so people looking at `#[serde(with = "crate::serde_impl::_"]` attribute don't have to check the implementation to know what's going on.